WordPress/Classes/Networking/ReaderTopicServiceRemote.m

```diff
@@ -234,9 +236,18 @@ - (RemoteReaderSiteInfo *)siteInfoForFeedResponse:(NSDictionary *)response
     if (![siteInfo.siteName length] && [siteInfo.siteURL length] > 0) {
         siteInfo.siteName = [[NSURL URLWithString:siteInfo.siteURL] host];
     }
+    NSString *endpointPath = [NSString stringWithFormat:@"read/feed/%@/posts/", siteInfo.feedID];
+    siteInfo.path = [self endointForPath:endpointPath];
     return siteInfo;
 }
 
+- (NSString *)endointForPath:(NSString *)endpoint
+{
+    NSString *path = [self pathForEndpoint:endpoint withVersion:ServiceRemoteRESTApiVersion_1_2];
```

Comment:
Pretty minor, but the naming around this is a bit weird: there's a `...ForPath` parameter named `endpoint`, which it's then turned into a `path`. How about:

```objc
- (NSString *)endpointUrlForPath:(NSString *)path
{
    NSString *absolutePath = [self pathForEndpoint:path withVersion:ServiceRemoteRESTApiVersion_1_2];
    NSURL *url = [NSURL URLWithString:absolutePath relativeToURL:self.api.baseURL];
    return [url absoluteString];
}
```